deploy-image/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-alpine
+FROM python:3.7-alpine
```

Comment:
Intentional?

Reply:
Yes, setuptools had an issue with the previous version. (see i ensure setuptools is up todate before the build command as well.
./build.sh was failing for version 3.8-alpine